evennia/server/server.py

```diff
@@ -151,7 +151,8 @@ def __init__(self, application):
         sys.path.insert(1, '.')
 
         # create a store of services
-        self.services = service.IServiceCollection(application)
+        self.services = service.MultiService()
+        self.services.setServiceParent(application)
```

Comment:
Oh I forgot about this part. When I was tinkering with services to see if there was anything wrong there, I just changed that to mirror a suggestion in some stack overflow post about threadpools hanging. I think it's equivalent code and doesn't make a difference, but didn't seem to hurt anything anyway.